src/components/ViewCourse/sections/ViewCourseInstructors.js

```diff
@@ -108,10 +102,8 @@ const ViewCourseInstructors = (props) => {
           {ariaLabel => (
             <Button
               aria-label={ariaLabel}
-              buttonStyle="primary"
```

Comment:
Lovely to see all these `buttonStyle="primary"` and `marginBottom0` attributes going!